tests/test_ewmh.py

```diff
@@ -124,6 +124,38 @@ def test_wm_state_type(hlwm, x11):
     assert len(prop.value) == 2
 
 
+def test_ewmh_focus_client(hlwm, x11):
+    hlwm.call('set focus_stealing_prevention off')
+    # add another client that has the focus on the other tag
```

Comment:
There is no other tag?

Reply:
I've fixed it, thanks! (And I've also renamed the variables)

Reply:
Thanks! Can you maybe update the variables in the second testcase, too?

Reply:
I've renamed it in PR #882.

Reply:
Alright, thanks!